chain/chunks/src/logic.rs

```diff
@@ -158,47 +158,37 @@ pub fn decode_encoded_chunk(
     shard_tracker: &ShardTracker,
 ) -> Result<(ShardChunk, PartialEncodedChunk), Error> {
     let chunk_hash = encoded_chunk.chunk_hash();
-    let _span = debug_span!(
+    let span = debug_span!(
         target: "chunks",
         "decode_encoded_chunk",
         height_included = encoded_chunk.cloned_header().height_included(),
         shard_id = ?encoded_chunk.cloned_header().shard_id(),
```

Comment:
```suggestion
        shard_id = ?encoded_chunk.cloned_header().shard_id(),
        encoded_length = tracing::field::Empty,
        num_tx = tracing::field::Empty,
        me = me.map(tracing::field::DebugValue),
```

Could also use `?me` for `me` field, but the suggested change will only show/emit `me` when it is `Some`. And that's just a little bit cleaner overall.